daemon/api_test.go

```diff
@@ -828,6 +828,112 @@ func (s *apiSuite) TestSnapInfoIgnoresRemoteErrors(c *check.C) {
 	c.Check(rsp.Result, check.NotNil)
 }
 
+func (s *apiSuite) TestMapLocalFields(c *check.C) {
+	media := snap.MediaInfos{
+		{
+			Type: "screenshot",
```

Comment:
AFAICT both screenshots are not used in this test, should they get tested somehow as well? Or can we remove them?

Reply:
we check that `mapLocal(about).Media` is equal to this `media` here, meaning the screenshots and icons made it through untouched. At the risk of seeming thick, how more would get them tested?

Reply:
Aha, ok - I misunderstood the purpose of the test, I removed the screenshots and the test still passed, this is why I was confused, now it does make more sense, thank you!